src/imgui/ImGuiUtils.cc

```diff
@@ -283,6 +284,30 @@ std::string getShortCutForCommand(const HotKey& hotkey, std::string_view command
 	return "";
 }
 
+std::optional<std::string> getKeyChordName(ImGuiKeyChord keyChord)
+{
+	int keyCode = ImGuiKey2SDL(ImGuiKey(keyChord & ~ImGuiMod_Mask_));
+	if (keyCode != SDLK_UNKNOWN) {
+		auto scanCode = SDL_GetScancodeFromKey(keyCode);
+		return strCat((keyChord & ImGuiMod_Ctrl ? "Ctrl+" : ""),
+			(keyChord & ImGuiMod_Shift ? "Shift+" : ""),
+			(keyChord & ImGuiMod_Alt ? "Alt+" : ""),
+			(keyChord & ImGuiMod_Super ? (ImGui::GetIO().ConfigMacOSXBehaviors ? "Cmd+" : "Super+") : ""),
+			SDL_GetScancodeName(scanCode));
+	}
+	return {};
+}
+
+std::optional<ImGuiKeyChord> getKeyChordValue(const std::string_view name)
+{
+	ImGuiKeyChord keyMods = {};
+	keyMods |= StringOp::containsCaseInsensitive(name, "Ctrl+")  ? ImGuiMod_Ctrl  : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Shift+") ? ImGuiMod_Shift : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Alt+")   ? ImGuiMod_Alt   : 0;
+	SDL_Keycode keyCode = SDL_GetKeyFromName(std::string(name).c_str());
```

Comment:
Does `SDL_GetFromName()` work correctly when `name` still contains suffixes like `+Ctrl`?

Reply:
It's fixed now.